ocs/agent/aggregator.py

```diff
@@ -64,7 +64,7 @@ def make_filename(base_dir, make_subdirs=True):
             raise FileNotFoundError("Subdir {} does not exist"
                                     .format(subdir))
 
-    time_string = start_datetime.strftime("%Y-%m-%d-%H-%M-%S")
+    time_string = int(start_datetime.timestamp())
```

Comment:
You don't need the start_datetime variable any more.

Reply:
Right, good point. Removed.